src/panel/layouts.js

```diff
@@ -25,6 +26,30 @@ export function getMapPaddings({ isMobile, isDirectionsActive }) {
   return MOBILE_CARD;
 }
 
+export function getVisibleBbox() {
```

Comment:
By passing the MapBoxGL map instance as parameter to this function, instead of using it multiple times as `window.map.mb`, you make this function decoupled from the `scene.js` implementation and from global variables, which is always a good thing. It's easier to maintain and to unit test in isolation.